include/swift/RemoteInspection/DescriptorFinder.h

```diff
@@ -39,13 +41,69 @@ struct BuiltinTypeDescriptorBase {
   virtual llvm::StringRef getMangledTypeName() = 0;
 };
 
+/// An abstract interface for a field record.
```

Comment:
What is a FieldRecord versus a FieldDescriptor?

Reply:
A field descriptor describes a nominal type, it has an array of field records, which describes that type's fields.